Makefile

```diff
@@ -1,3 +1,8 @@
+CONTAINER_ENGINE?=$(shell docker version >/dev/null 2>&1 && echo docker)
+ifeq ($(CONTAINER_ENGINE), "")
+	CONTAINER_ENGINE==$(shell podman version >/dev/null 2>&1 && echo podman)
+endif
+
 GIT_COMMIT?="$(shell git rev-parse HEAD | head -c 7)"
 NAME_POSTFIX?="$(shell docker ps -a | wc -l | xargs)"
```

Comment:
Not sure that's going to work with podman.

Reply:
We verified it in an env with podman.